octobot_trading/exchanges/connectors/ccxt/ccxt_connector.py

```diff
@@ -308,7 +309,7 @@ async def get_open_orders(self, symbol: str = None, since: int = None,
         if self.client.has['fetchOpenOrders']:
             with self.error_describer():
                 return [
-                    self.adapter.adapt_order(order)
+                    self.adapter.adapt_order(order, symbol=symbol)
```

Comment:
Why not using order.symbol in adapt_order?

Reply:
order is not parsed yet and when it's an async-created order on the exchange side, we only have its id, not even the symbol